src/SteamAPI.ts

```diff
@@ -335,46 +336,50 @@ export default class SteamAPI {
 	async getGameDetails(
 		app: number | number[],
 		{ language = this.language, currency = this.currency, filters = [] } = {}
-	): Promise<{ [key: string]: any }> {
```

Comment:
Rereading this again and definitely the original type here is always an object mapping ID -> details. I think this is cleaner to do anyways! Having a single return type is nicer to implement using this function IMO.